### ISSUE TITLE
Docs: serverExternalPackages doesn't exist/isn't valid

### ISSUE THREAD

**Issue by Clonkex** [vercel/next.js#70728]
### What is the documentation issue?

[This page](https://nextjs.org/docs/app/api-reference/next-config-js/serverExternalPackages) says to use `serverExternalPackages` to exclude packages from bundling, but this results in an error:

```
 ✓ Starting...
 ⚠ Invalid next.config.mjs options detected:
 ⚠     Unrecognized key(s) in object: 'serverExternalPackages'
 ⚠ See more info here: https://nextjs.org/docs/messages/invalid-next-config
 ```

### Is there any context that might help us understand?

I was trying to use Sequelize, but it didn't work while being bundled. Thankfully there is an [SO answer](https://stackoverflow.com/a/79029968/2288578) that showed how to do it now:

```
/** @type {import('next').NextConfig} */
const nextConfig = {
    webpack: (config, { isServer }) => {
        if (isServer) {
            // Exclude packages from the server-side bundle
            config.externals.push('sequelize');
        }
        return config;
    },
}

export default nextConfig;
```

### Does the docs page already exist? Please link to it.

https://nextjs.org/docs/app/api-reference/next-config-js/serverExternalPackages

**Comment by nadir** [comment]
I can confirm another page is also missing, https://nextjs.org/docs/app/building-your-application/optimizing/package-bundling

**Comment by samcx** [comment]
@Clonkex Can you clarify which version you are using?

**Comment by Clonkex** [comment]
@samcx I'm on 14.2.14. However, looking at the linked page now, I see a different page to what I saw when I opened this issue. If I switch the documentation version to 15.0.0 Canary, I see the page I previously saw. You might think I had simply accidentally set it to Canary, but a) I'm 99% sure I didn't have it set to Canary, b) I copy-pasted the URL directly from the page I had been looking at (and the URL doesn't include the canary part), and c) this [Wayback Machine snapshot](https://web.archive.org/web/20240917102404/https://nextjs.org/docs/app/api-reference/next-config-js/serverExternalPackages) seems to confirm that it was displaying the 15.0.0 Canary page on the stable URL.

I guess since the issue appears to have been resolved, this issue can be closed.
